test/predictedspeeds.cc

```diff
@@ -63,7 +52,11 @@ TEST(PredicteSpeeds, test_decoding) {
 
   // Decode the base64 string and cast the data to a raw string of signed bytes
   auto decoded_str = decode64(encoded_speed_string);
-  EXPECT_EQ(decoded_str.size(), 402);
+  // HACK(mookerji): kDecodedSpeedSize+1 is the expected size (as opposed to kDecodedSpeedSize)
+  // because we start reading from a 1-byte offset below at the little endian conversion. This is
+  // actually a broken test fixture because we start reading from 0 in the actual CLI decoding in
+  // src/mjolnir/valhalla_add_predicted_traffic.cc
```

Comment:
to fix this should we do a full round trip? we could loop over the values in the block of data below converting them to big endian, pushing back 2 bytes at a time into a string, encode the whole string, and then run the test as is but start the index at 0 instead of 1. i cant understand why this is written this way otherwise?